detector/detector.go

```diff
@@ -263,17 +268,27 @@ func (d *Detector) evalToStringTokens(t token.Token) ([]token.Token, error) {
 	return tokens, nil
 }
 
-func (d *Detector) isSkip(deepCheck bool, target string) bool {
+func (d *Detector) isSkip(deepCheck bool, targetType string, target string) bool {
 	if deepCheck && !d.Config.DeepCheck {
 		return true
 	}
 
-	targetResources := 0
-	for _, template := range d.Templates {
-		targetResources += len(template.Node.(*ast.ObjectList).Filter("resource", target).Items)
+	targets := 0
+	switch targetType {
+	case "resource":
+		for _, template := range d.Schema {
+			targets += len(template.FindResources(target))
+		}
+	case "module":
```

Comment:
__[gometalinter]__
* [goconst] 2 other occurrence(s) of "module" found in: test_helper.go:82:7 detector.go:213:7

<!-- Comment by [SideCI](https://sideci.com) -->